### ISSUE TITLE
Graph in View repeat Panel disappear after change dashboard variables

### ISSUE THREAD

**Issue by okhowang** [grafana/grafana#21455]
**What happened**:
when view repeated panel, changing variable will make graph disappeared

**What you expected to happen**:
all repeated panel view as first repeat panel

**How to reproduce it (as minimally and precisely as possible)**:
there is a dashboard which has a repeat panel graph,
![image](https://user-images.githubusercontent.com/3352585/72236240-fcfc6f00-3610-11ea-9884-4acb13e48119.png)
after entering view mode of a repeated panel which should not be edited
![image](https://user-images.githubusercontent.com/3352585/72236249-038ae680-3611-11ea-9861-9a2863f78899.png)
everything is ok.
but after change variables, graph disappeared.
![image](https://user-images.githubusercontent.com/3352585/72236253-0b4a8b00-3611-11ea-8676-3ef683ffcad6.png)

but everything is ok when changing variables in first repeat panel which should be edit

**Anything else we need to know?**:

**Environment**:
- Grafana version: 6.5.0
- Data source type & version: prometheus
- OS Grafana is installed on: linux
- User OS & Browser: chrome
- Grafana plugins: N/A
- Others:

**Comment by okhowang** [comment]
it seem like
```typescript
  startRefresh() {
    //when here, if current panel is repeated panel, panel.fullscreen will be cleared
    this.events.emit(PanelEvents.refresh);

    for (const panel of this.panels) {
      if (!this.otherPanelInFullscreen(panel)) {
        panel.refresh();
      }
    }
  }
 //...

  otherPanelInFullscreen(panel: PanelModel) {
    return this.meta.fullscreen && !panel.fullscreen;
  }
```

**Comment by torkelo** [comment]
Yes, this is not supported, bit tricky. If you go full screen view of a repeated panel Grafana should lock the variables so you cannot change them (if they are the repeated variable or it depends on it)
